Feature/integration project frontend 02

pdrflp0 opened the pull request.

pdrflp0 committed:
### Changed
   - The model logic is now used being imported from the back-end project.
   - The lines logic is now used being imported from the back-end project.
   - Screen layout adjusted to cover the unnecessary "spaces" that existed.
   - Unnecessary comments.

### Removed
   - Model logic code.
   - Lines logic code.

pdrflp0 committed:
### Changed
   - Comments in Portuguese to English.

### Removed
   - Unnecessary comments.
   - Part of useless codes

Bacuriim commented on src/main/java/br/eletra/frontend/Main.java:
Essas linhas de inicialização de modelos são inúteis, por favor remover.

pdrflp0 replied:
Pendência resolvida.

Bacuriim commented on src/main/java/br/eletra/frontend/controllers/ScreenController.java:
```diff
@@ -20,27 +20,16 @@ public class ScreenController {
-//        ObservableList<String> lines = FXCollections.observableArrayList(
-//                "Cronos",
-//                "Ares"
-//        );
```
Essa parte comentada do código pode ser removida, pois não tem função

pdrflp0 replied:
Pendência resolvida.

Bacuriim reviewed (changes requested):
Resolva essas pendencias e me marque no cartão que irei recorrigir!

pdrflp0 committed:
### Added
   - Plugin in pom.xml for the project's ".jar" file to function correctly, independently.

### Removed
   - Part of the model initialization code in Main.java.

Bacuriim reviewed (approved):
Aprovado!
